lib/network/http2_network_test.go

```diff
@@ -11,9 +11,35 @@ import (
 	"boscoin.io/sebak/lib/common"
 	"github.com/stellar/go/keypair"
 	"github.com/stretchr/testify/assert"
+	"math/rand"
+	"net"
+	"strconv"
+	"sync"
+	"time"
 )
 
+var testPort = "5000"
+var once sync.Once
+
+func getPort() string {
+	once.Do(func() {
+		for {
+			s := rand.NewSource(int64(time.Now().Nanosecond()))
+			r := rand.New(s)
+			testPort = strconv.Itoa(r.Intn(65535))
+
+			ln, err := net.Listen("tcp", ":"+testPort)
```

Comment:
I think, this checking part is good :)

Reply:
In terms of information, the range of port can be changed. As we know, 0 ~ 1023 is well-known ports and from 1024 to 49151 is also registered ports(https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Well-known_ports), so we can use the private port range to escape the conflict.(https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Dynamic,_private_or_ephemeral_ports)

Reply:
use ephemeral ports
range 49152 ~ 65535
@spikeekips